react/react1/week3/time-registration/src/ShiftOverview.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState} from 'react'
+import "./ShiftOverview.css"
+import ShiftInfo from './ShiftInfo'
+
+function ShiftOverview({allUsers}) {
```

Comment:
I would have liked to be able to see all the users at same time.
Also, maybe clear the input fields when the user is submitted